stock_lock_lot/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,4 @@
+This module allows you to define whether a Serial Number/lot is blocked
```

Comment:
IMHO, 'blocked' can have several meanings and should be clarified. When and how is a lot blocked ?